Week3/homework/CreditCardValidate.js

```diff
@@ -0,0 +1,46 @@
+"use strict"
+
+function validCreditCard (num) {
+    let card = num;
+    let cardNumber = card.split("");
+    let cardNumberDif = cardNumber.filter((este, i) => cardNumber.indexOf(este) === i);
+    let cardNumberLen = cardNumberDif.length;
+    let soma = 0;
+         /* My intention here is to define that the digits can't be all the same*/
+        if (cardNumberLen = 1) {
```

Comment:
Ah. Be careful here please. Ideally, we should not be doing any assignment operation i.e `cardNumberLen = 1` inside an if condition as it leads to unexpected results. Perhaps you wanted to do `cardNumberLen === 1` instead?